.github/workflows/netsim_integration.yml

```diff
@@ -84,3 +84,6 @@ jobs:
         d=$(cat report_metro.txt)
         metro_data=$(printf "%s\n " "$d")
         echo "$metro_data" 
+
+
+        arminhajro@gmail.com
```

Comment:
?

Reply:
lol some bad paste from somewhere (it's my electricians email I think xD)